Dockerfile

```
@@ -0,0 +1,61 @@
+
+WORKDIR /app
+
+# Copy package files
+COPY package.json package-lock.json ./
+
+# Set up GitHub token for private npm packages
+ARG GITHUB_TOKEN
+RUN echo "//npm.pkg.github.com/:_authToken=${GITHUB_TOKEN}" >> .npmrc
+
+# Install only production dependencies
+RUN npm ci --only=production
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)